flink-ml-iteration/src/main/java/org/apache/flink/iteration/operator/perround/AbstractPerRoundWrapperOperator.java

```diff
@@ -223,7 +246,44 @@ public void initializeState(StreamTaskStateInitializer streamTaskStateManager)
 
     @Override
     public void initializeState(StateInitializationContext context) throws Exception {
-        // Do thing for now since we do not have states.
+        parallelismState =
```

Comment:
Do we need to call the super method?

Reply:
Perhaps this is not necessary since the super is empty?

Reply:
But I think it might be some potential bug, right? So I prefer to call it even if the super is empty WDYT?

Reply:
I also agree we'd better always call super method, but when I add the code it suddenly comes to the that the operator does not extend the `AbstractStreamOperator`, thus in this case we do not have the super method to call~

Reply:
thanks for your explaination!